### PR TITLE
Event about component #128

### PR DESCRIPTION
**Pull request by momanyisamuel**
### Contributor checklist

- [] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

Added missing elements for CardAbout.vue

### Related issue

-  #128 
- #129

### CODE REVIEW

File: frontend/components/Card/CardAbout.vue [hunk 1]
```diff
@@ -1,11 +1,38 @@
...
+      <div class="flex justify-start items-center gap-3 mb-3">
+        <h3 class="text-left responsive-h3 font-display">About</h3>
+        <Icon name="bi:pencil-square" size="1.2em" />
+      </div>
+      <button
+        class="px-2 py-1 border-2 border-light-cta-orange bg-[#F1993D4D] text-light-cta-orange rounded-2xl flex items-center gap-2 mb-3"
+      >
+        <Icon name="bi:globe" size="1.2em" />
+        Housing and Community
+      </button>
```
Comment (Mradosi) [inline frontend/components/Card/CardAbout.vue:16]: I suggest using the TopicMarker component instead, as it is already created and styled and used in other places.

Comment (momanyisamuel) [reply frontend/components/Card/CardAbout.vue:16]: oh cool thanks, will make the changes, still working on this for #129

Comment (andrewtavis) [reply frontend/components/Card/CardAbout.vue]: Thank you, @Mradosi!! :)

Comment (momanyisamuel) [reply frontend/components/Card/CardAbout.vue]: @andrewtavis @Mradosi  Added requested changes above and some more changes for #129

Comment (andrewtavis) [reply frontend/components/Card/CardAbout.vue]: Totally fine with updating Tailwind, @momanyisamuel :) Do you want to send that change along as well?

Comment (momanyisamuel) [reply frontend/components/Card/CardAbout.vue]: done 👍

**Review by andrewtavis — APPROVED** [review]
Thanks for all this, @momanyisamuel! I think that we may need to hold off on `line-clamp` for now as it might not be available in [Nuxt Tailwind](https://tailwindcss.nuxtjs.org/), but let's keep it in mind for later :) I'll investigate, and we can invest some more time into this later when we're closer to beta 😊

**Pull request merged**